session/manager.go

```diff
@@ -55,6 +58,27 @@ type publisher interface {
 	Publish(topic string, data interface{})
 }
 
+// KeepAliveParams contains keep alive options.
+type KeepAliveParams struct {
+	SendInterval    time.Duration
+	MaxSendErrCount int
+}
+
+// Params contains common configuration options for session manager.
+type Params struct {
+	KeepAlive KeepAliveParams
+}
+
+// DefaultParams returns default params.
+func DefaultParams() Params {
+	return Params{
+		KeepAlive: KeepAliveParams{
```

Comment:
We have the same values in the connection manager. Can we have it defined in a single place?

Reply:
For now values are the same, but what if you decide to change it for provider side etc. ? They are also in different packages so for simplicity I would leave it as is.